pkg/client/s3client.go

```diff
@@ -48,9 +48,8 @@ const (
 func NewS3Client(c *Client, instanceName, region string) (s3client *S3Client, err error) {
 	s3client = &S3Client{}
 	var instanceID string
-	svcs, err := c.ResourceClient.ListInstances(controllerv2.ServiceInstanceQuery{
+	svcs, err := c.ResourceClientV2.ListInstances(controllerv2.ServiceInstanceQuery{
 		Type: "service_instance",
-		Name: instanceName,
```

Comment:
As discussed, please look at this line deletion and any other merge override issues.